documentation/src/main/asciidoc/topics/proc_configuring_server_keystores.adoc

```diff
@@ -0,0 +1,37 @@
+[id='configuring-server-keystores_{context}']
+= Configuring {brandname} Server Keystores
+Add keystores to {brandname} Server and configure it to present SSL/TLS certificates that verify its identity to clients.
+If a security realm contains TLS/SSL identities, it encrypts any connections to {brandname} Server endpoints that use that security realm.
+
+.Prerequisites
+
+* Create a keystore that contains certificates, or certificate chains, for {brandname} Server.
+
+{brandname} Server supports the following keystore formats: JKS, JCEKS, PKCS12,
+BKS, BCFKS, and UBER.
+
+[IMPORTANT]
+====
+In production environments, server certificates should be signed by a trusted Certificate Authority, either Root or Intermediate CA.
+====
+
+.Procedure
+
+. Add the keystore that contains SSL/TLS identities for {brandname} Server to the `{server_home}/server/conf` directory.
+. Add a `server-identities` definition to the {brandname} Server security realm.
+. Specify the keystore file name with the `path` attribute.
+. Provide the keystore password and certificate alias with the `keystore-password` and `alias` attributes.
+
+.{brandname} Server keystore configuration
+[source,xml,options="nowrap",subs=attributes+]
+----
+include::config_examples/server_ssl_identity_truststore.xml[]
+----
+
+.Next steps
+
+Add the keystore to client configuration as a trust store so that clients can verify the {brandname} Server identity.
```

Comment:
is it supposed to add *the* server Keystore or just create a Keystore with the CA?

Reply:
good distinction. I believe it should be a keystore with the CA. I've reworded this. think Ryan also asked a similar question here: https://github.com/infinispan/infinispan/pull/9354#discussion_r655491834